syntax = "proto3";

package million.config.ss;

import "module/module_id.proto";

enum ServiceNameId {
    option (module.module_id) = MODULE_ID_CONFIG;

    SERVICE_NAME_ID_INVALID = 0;
    SERVICE_NAME_ID_CONFIG = 1;
}
